requirements.txt

```diff
@@ -1,3 +1,12 @@
-datajoint>=0.13.0
-element-interface>=0.4.0
-plotly
+datajoint==0.14.0
+element_interface==0.5.0
```

Comment:
```suggestion
element-interface>=0.4.0
```

Reply:
Let's keep the previous version as the code can run with `element-interface>=0.4.0`.